src/main/scala/org/scoverage/coveralls/CIService.scala

```diff
@@ -25,18 +25,22 @@ case object TravisPro extends CIService {
 }
 
 case object GitHubActions extends CIService {
-  val name = "github"
+  val name = ""
```

Comment:
Apparently, that was the reason it was failing. Idk, maybe in case of `gihub actions` some more params are required by coveralls.io api.

Reply:
No, params are fine - it's failing because when the service is "github" you need to pass a GitHub token, not a token obtained in Coveralls pages.

Reply:
Ah, that makes sense, good catch!